BZAccessControl/contracts/BZAccessControl.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/utils/Context.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+import "./interfaces/IBZAccessControl.sol";
+import "./interfaces/IBunzz.sol";
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+
+contract BZAccessControl is Context, Ownable, IBZAccessControl, IBunzz, ERC165 {
```

Comment:
Missing NatSpec

Reply:
Update the smart contract and added NatSpec